### DISCUSSION TITLE
Does Aspire support or plan to support multiple resource groups?

### DISCUSSION THREAD

**Discussion by jonoabroad** [microsoft/aspire#3872] [Q&A] [answered]
My interest is primarily in deployment.

This seems some what related to https://github.com/dotnet/aspire/issues/521, there was another "thing" I can not for the life of me find it, which linked to the issue. It suggested allowing one to group 'like' containers.

I want to be able to have some of my containers deployed to their own resource group.  As an example - deploying telemetry containers to their own resource group.

**Comment by davidfowl** [comment, marked as answer]
Not for GA no. Customization of the compute layer will come much later after GA. We’re working through the design issues to make it possible in the future

**Reply by jonoabroad** [reply]
Thank you, I thought this was the case, but wanted to make sure.

**Reply by davidfowl** [reply]
You can use azd infra synth to spit out the infrastructure and tweak things... This is a short term solution though...

**Reply by jonoabroad** [reply]
Thank you, short term is fine for the moment.
